src/mscorlib/shared/System/Runtime/InteropServices/MemoryMarshal.cs

```diff
@@ -40,9 +40,14 @@ public static bool TryGetArray<T>(ReadOnlyMemory<T> readOnlyMemory, out ArraySeg
         }
 
         /// <summary>
-        /// Get a <see cref="OwnedMemory{T}"/> from the underlying memory.
-        /// If unable to get the <typeparamref name="TOwner"/>, return false with a default <typeparamref name="TOwner"/>.
+        /// Gets a <see cref="OwnedMemory{T}"/> from the underlying readOnlyMemory.
```

Comment:
As long as we're fixing them, it should be "Gets an" rather than "Gets a".

Reply:
Updated in https://github.com/dotnet/coreclr/pull/16479; have to make a change to remove the owner flag from the index